appveyor.yml

```diff
@@ -96,7 +96,8 @@ build_script:
 - cd %ROOT_DIRECTORY%
 - mkdir build && cd build
 - if defined cmake (
-  cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="/WX" -G "Visual Studio 14 2015 Win64" .. &&
+  set "PATH=c:\Python35-x64;%PATH%" &&
+  cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="/WX" -Dpython-bindings=%python% -Dboost-python-module-name="python" -G "Visual Studio 14 2015 Win64" .. &&
```

Comment:
`-DCMAKE_BUILD_TYPE=Release` makes no sense with the Visual Studio generator. To run release build we have to either run build using msbuild invocation and specify the configuration name, or generate makefiles.

Reply:
Do you mind linking documentation to such information (that `-DCMAKE_BUILD_TYPE=Release` does not make sense here)?

Reply:
https://cmake.org/cmake/help/v3.11/variable/CMAKE_BUILD_TYPE.html

Reply:
thanks

Reply:
I’m happy to help